packages/web3-eth-contract/src/log_subscription.ts

```diff
@@ -79,7 +79,7 @@ import { EventLog, ContractAbiWithSignature } from './types.js';
 export class LogsSubscription extends Web3Subscription<
 	{
 		error: Error;
-		connected: number;
+		connected: string;
 		data: EventLog;
```

Comment:
intersection type using existing `CommonSubscriptionEvents`,  instead of redefining these fields?

Reply:
Sorry, this `connected` should be removed from here, along with `error`, because its related codes are in the base class. And I had done that just now.
However, regarding `connected`, it was not used, and there was no code related to it, in any place in this class or in the base class.